src/components/DialoguesList/__tests__/__snapshots__/DialoguesList.test.jsx.snap

```diff
@@ -24,11 +26,15 @@ exports[`DialoguesList produces correct snapshot of multiple messages 1`] = `
     <div
       className="bubble me bubble-img"
     >
-      <img
-        alt="imageMessage"
-        className="content-img"
-        src="https://play-lh.googleusercontent.com/zepWGavYYErAIBXFZb6OT14I6b-m4TyaG3yjqZy6Hnsmi64vL3upQ3KUsV6Wnsm-e9M=w512"
-      />
+      <p
```

Comment:
Ideally would replace this with the Typography import from MUI :)

Reply:
This file is generated snapshot after running npm test, I don't think I can change this

Reply:
Lmao sorry brain just blanked on what file I was looking at